src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -347,14 +354,14 @@ export default {
 		@after-leave="onAfterLeave">
 		<aside id="app-sidebar-vue" class="app-sidebar">
 			<header :class="{
-					'app-sidebar-header--with-figure': hasFigure,
+					'app-sidebar-header--with-figure': isSlotPopulated($slots?.header?.()) || background,
```

Comment:
`$slots` is guaranteed to not be null, it is at least an empty object, null check is redundant here and in other places in the component.

Reply:
Yeah, I wasn't sure it is guaranteed to be present and wanted to be safe. Fixed it.